src/pages/Transportation/Register/FourteenInnocents.tsx

```diff
@@ -1,3 +1,41 @@
+import { Header } from "../../../components/header";
+import fourteen1 from "../../../assets/Transportation/register/fourteen_1-min.png";
+import fourteen2 from "../../../assets/Transportation/register/fourteen_2.webp";
+import { RegisterCard } from "../../../components/Rigester_card";
+
+// import img .
+
+
+
 export function FourteenInnocents() {
-  return <div>FourteenInnocents</div>;
+  return (
+    <>
+      <Header heading="fourteen innocents" img="headerBg" />
+      <section className="flex gap-7">
+        
+        <RegisterCard
```

Comment:
please make an array of objects for the data. like

const FourteenInnocentsData = [
{
heading: "add heading",
img: "add image",
description: "details about card",
date: " ",
date1: " ",
date2: " ",
price: " ",
rate: " ",
button: " ",
}
]

and then map this array render the RegisterCard component.